mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -1105,6 +1108,59 @@ private[python] class PythonMLLibAPI extends Serializable {
     Statistics.kolmogorovSmirnovTest(data, distName, paramsSeq: _*)
   }
 
+  /**
+   * Wrapper around DistributedMatrices.rowMatrix factory method.
+   */
+  def createRowMatrix(rows: JavaRDD[Vector], numRows: Long, numCols: Int): RowMatrix = {
+    DistributedMatrices.rowMatrix(rows.rdd, numRows, numCols)
```

Comment:
I'm wondering out aloud if it would be good to have a Javafriendly constructor in `class RowMatrix` as in many other places.